doc/radosgw/cloud-transition.rst

```diff
@@ -311,10 +310,6 @@ For example:
   ]
 
 
-   radosgw-admin zonegroup placement rm --rgw-zonegroup default \
-                                          --placement-id default-placement \
-                                          --storage-class CLOUDTIER
-
```

Comment:
Is it your intent to remove this completely?

Reply:
Yes, this is duplicate of command in lines 293-296 and was a result of my botched rebase.

Reply:
I botch like 90% of the rebases I attempt, so it's not just you ;)